src/library/Help/Wrappers.ts

```diff
@@ -78,6 +78,7 @@ export const ContentWrapper = styled.div`
   h1 {
     font-family: 'NeueBit-R', 'sans-serif', sans-serif;
     margin-bottom: 1.75rem;
+    font-size: 29px;
```

Comment:
Hmm, IDK how that results in font-size 40px but that's what the browser dev tools say:

![Screenshot at 2023-06-20 19-00-41](https://github.com/gluwa/creditcoin-staking-dashboard/assets/1002300/32220626-9d9a-4566-9217-72a5a62fb9b9)